src/block-management/block-manager.scss

```diff
@@ -56,4 +57,20 @@
 	bottom: 0;
 	right: 0;
 	left: 0;
+}
+
+.blockHolderFocusedSemiBordered {
+	border-color: $gray-lighten-30;
```

Comment:
Would it make sense to use css `box-shadow` to apply these focus states? In theory that should prevent any shifting, just applying a shadow on the outside edge of the element.

Reply:
Unfortunately box-shadow property is not available in React Native and as I investigated it is not a very straight forward change to replace borders with shadows in RN. Because Android and iOS have very different methods for doing it and both methods have different side effects that needs to be addressed.

iOS has these props:
https://facebook.github.io/react-native/docs/shadow-props#shadowcolor

Android has the elevation option but there are no other options to fine-tune the shadow effect. So I am not even sure it'll give the results we want.
https://facebook.github.io/react-native/docs/view-style-props#elevation

I tried to apply these quickly but it messed up the child views' styles in different ways, so it didn't look like an ideal last minute change, but, if you want we can open a separate issue for this and try later? 

In this PR, I fixed the shifting of Android blocks. I don't expect any shifting will happen in cases where the block width is equal to window width, because in that case we don't have left&right borders at all.